Sources/Misc/Deprecations.swift

```diff
@@ -420,6 +420,19 @@ public extension Configuration.Builder {
         return self
     }
 
+    /**
+     * Set `observerMode`.
+     * - Parameter observerMode: Set this to `true` if you have your own IAP implementation and want to use only
+     * RevenueCat's backend. Default is `false`.
+     *
+     * - Note: This assumes your IAP implementation uses StoreKit 1.
```

Comment:
```suggestion
     * - Warn: This assumes your IAP implementation uses StoreKit 1.
```

Can't remember if `Warn` is the exact keyword, but I'm shooting for the yellow warning sign

Reply:
Oh yeah. It's warning, I'll change it:
```suggestion
     * - Warning: This assumes your IAP implementation uses StoreKit 1.
```